webapps/ui/cockpit/src/modules/components/LoadingIndicator/LoadingIndicatorSmall.js

```diff
@@ -15,13 +15,14 @@
  * limitations under the License.
  */
 
-module.exports = `<!-- # CE - camunda-bpm-webapp/webapp/src/main/resources-plugin/base/app/views/processInstance/job-retry-bulk-action.html -->
-<a class="btn btn-default btn-toolbar"
-   href
-   ng-click="openDialog()"
-   uib-tooltip="{{ 'PLUGIN_JOB_RETRY_BULK_RETRIES_FAILED_JOB' | translate }}"
-   tooltip-placement="left">
-  <span class="glyphicon glyphicon-repeat"></span>
-</a>
-<!-- / CE - camunda-bpm-webapp/webapp/src/main/resources-plugin/base/app/views/processInstance/job-retry-bulk-action.html -->
-`;
+import React from "react";
+
+import "./LoadingIndicator.scss";
+
+export default function LoadingIndicatorSmall() {
```

Comment:
Maybe we can add an optional size prop to the existing `<LoadingIndicator>` Component instead of creating a new one?

The `.scss` file should follow the Naming convention (`LoadingIndicatorSmall.scss`) or be included in the `index.js`. I see this as an indicator that these components should be merged.